blockchain/genesis.go

```diff
@@ -199,6 +199,9 @@ func SetupGenesisBlock(db database.DBManager, genesis *Genesis, networkId uint64
 		db.WriteChainConfig(stored, newcfg)
 		return newcfg, stored, nil
 	} else {
+		if storedcfg.Governance == nil {
+			logger.Crit("Governance field is missing")
```

Comment:
How about `Failed to read governance. ChainConfig.Istanbul == nil. storedcfg.Governance == nil`? :)

Reply:
Fixed.